README.md

```diff
@@ -189,6 +189,21 @@ Compares experiment settings accros development/staging/production environments.
 
     biomage experiment compare my-experiment-id
 
+#### experiment download
+
+Download files associated with an experiment.
+
+    biomage experiment download -e my-experiment-id -i environment
+
+Currently download of the following files are supported:
```

Comment:
`is`, not `are` in `files are supported`

Reply:
Aah I see... `support` refers to `download`, which is singular, so it should be `is`